xain_fl/coordinator/coordinator.py

```diff
@@ -34,61 +34,74 @@ class Coordinator:  # pylint: disable=too-many-instance-attributes
-            during initialisation, then :class:`~.FederatedAveragingAgg` in used.
-        DEFAULT_CONTROLLER (:class:`~.Controller`): if no Controller instance is provided
-            during initialisation, then :class:`~.RandomController` in used.
-
-    Attributes:
-        num_rounds (:obj:`int`, optional): The number of rounds of the training
-            session. Defaults to 10.
-        minimum_participants_in_round (:obj:`int`, optional): The minimum number of
-            participants that participate in a round. Defaults to 1.
-        fraction_of_participants (:obj:`float`, optional): The fraction of total
-            connected participants to be selected in a single round. Defaults to 1.0,
-            meaning that all connected participants will be selected.
-            It must be in the (0.0, 1.0] interval.
-        weights (:obj:`list` of :class:`~numpy.ndarray`, optional): The weights of
-            the global model. Defaults to [].
-        epochs (:obj:`int`, optional): Number of training iterations local to
-            Participant.  Defaults to 0.
-        epochs_base (:obj:`int`, optional): Global number of epochs as of last
-            round. Defaults to 0.
-        aggregator: (:class:`~.Aggregator`, optional): The type of aggregation
-            to perform at the end of each round. Defaults to :class:`~.FederatedAveragingAgg`.
-        controller: (:class:`~.Controller`, optional): Controls how the Participants
-            are selected at the start of each round. Defaults to :class:`~.RandomController`.
-        """
+    Args:
+
+        num_rounds: The number of rounds of the training session
+
+        minimum_participants_in_round: The minimum number of
+            participants that participate in a round
+
+        fraction_of_participants: The fraction of total connected
+            participants to be selected in a single round. Defaults to
+            1.0, meaning that all connected participants will be
+            selected. It must be in the (0.0, 1.0] interval.
+
+        weights: The weights of the global model.
+
+        epochs: Number of training iterations local to Participant.
+
+        epochs_base: Global number of epochs as of last round.
+
+        aggregator: The type of aggregation to perform at the end of
+            each round. Defaults to :class:`~.FederatedAveragingAgg`.
+
+        controller: Controls how the Participants are selected at the
+            start of each round. Defaults to
+            :class:`~.RandomController`.
+
+    """
 
     DEFAULT_AGGREGATOR: Aggregator = FederatedAveragingAgg()
+    """if no Aggregator instance is provided during initialisation, then
+    :class:`~.FederatedAveragingAgg` in used.
+
+    """
+
     DEFAULT_CONTROLLER: Controller = RandomController()
+    """if no Controller instance is provided during initialisation, then
+:class:`~.RandomController` in used."""
```

Comment:
i would indent it under the above line and add """ on the line below

Reply:
Done (if I understood your comment correctly)